stl/inc/chrono

```diff
@@ -617,27 +669,1290 @@ namespace chrono {
+        return _Year / _Mdl;
+    }
+    _NODISCARD constexpr year_month_day_last operator/(const month_day_last& _Mdl, int _Year) noexcept {
+        return year{_Year} / _Mdl;
+    }
+    _NODISCARD constexpr year_month_weekday operator/(const year_month& _Ym, const weekday_indexed& _Wdi) noexcept {
+        return year_month_weekday{_Ym.year(), _Ym.month(), _Wdi};
+    }
+    _NODISCARD constexpr year_month_weekday operator/(const year& _Year, const month_weekday& _Mwd) noexcept {
+        return year_month_weekday{_Year, _Mwd.month(), _Mwd.weekday_indexed()};
+    }
+    _NODISCARD constexpr year_month_weekday operator/(int _Year, const month_weekday& _Mwd) noexcept {
+        return year{_Year} / _Mwd;
+    }
+    _NODISCARD constexpr year_month_weekday operator/(const month_weekday& _Mwd, const year& _Year) noexcept {
+        return _Year / _Mwd;
+    }
+    _NODISCARD constexpr year_month_weekday operator/(const month_weekday& _Mwd, int _Year) noexcept {
+        return year{_Year} / _Mwd;
+    }
+    _NODISCARD constexpr year_month_weekday_last operator/(const year_month& _Ym, const weekday_last& _Wdl) noexcept {
+        return {_Ym.year(), _Ym.month(), _Wdl};
+    }
+    _NODISCARD constexpr year_month_weekday_last operator/(
+        const year& _Year, const month_weekday_last& _Mwdl) noexcept {
+        return {_Year, _Mwdl.month(), _Mwdl.weekday_last()};
+    }
+    _NODISCARD constexpr year_month_weekday_last operator/(int _Year, const month_weekday_last& _Mwdl) noexcept {
+        return year{_Year} / _Mwdl;
+    }
+    _NODISCARD constexpr year_month_weekday_last operator/(
+        const month_weekday_last& _Mwdl, const year& _Year) noexcept {
+        return _Year / _Mwdl;
+    }
+    _NODISCARD constexpr year_month_weekday_last operator/(const month_weekday_last& _Mwdl, int _Year) noexcept {
+        return year{_Year} / _Mwdl;
+    }
+
+    // Calendrical constants
+    inline constexpr weekday Sunday{0};
+    inline constexpr weekday Monday{1};
+    inline constexpr weekday Tuesday{2};
+    inline constexpr weekday Wednesday{3};
+    inline constexpr weekday Thursday{4};
+    inline constexpr weekday Friday{5};
+    inline constexpr weekday Saturday{6};
+
+    inline constexpr month January{1};
+    inline constexpr month February{2};
+    inline constexpr month March{3};
+    inline constexpr month April{4};
+    inline constexpr month May{5};
+    inline constexpr month June{6};
+    inline constexpr month July{7};
+    inline constexpr month August{8};
+    inline constexpr month September{9};
+    inline constexpr month October{10};
+    inline constexpr month November{11};
+    inline constexpr month December{12};
+
+    // Returns the number of fractional digits of _Num / _Den in the range [0, 18].
+    // If it can't be represented, 6 is returned.
+    // Example: _Fractional_width(1, 8) would return 3 for 0.125.
+    constexpr unsigned int _Fractional_width(intmax_t _Num, const intmax_t _Den) {

```

Comment:
IILE'd so now impossible to discard :D

I've moved the comment to inside the function as otherwise the comment would come up in IntelliSense. Not sure as to what the policy is regarding that (if any); so just keeping it as it was with no visible comment.